src/platform/nxp/common/OTAImageProcessorImpl.cpp

```diff
@@ -22,6 +22,8 @@
 
 #include "OTAImageProcessorImpl.h"
 
+#define MIN_REBOOT_DELAY 200
```

Comment:
Add units to names to this like `_MS` or `_SEC`. Can this be a C++-style constexpr instead of a macro?